index.js

```diff
@@ -32,6 +54,14 @@ getFullNames(usersArray);
 
 const getUsersCreditDetails = arr => {
   // Your code goes here ...
+  const UsersCreditDetails=[];
+  
+  for (let user of arr) {
+    let {firstName,lastName,balance} =user
```

Comment:
In this case, you could have also used a const. There's no huge consensus online, but if you know you won't change it, a const considered "preferable"